src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/OffenderDetail.kt

```diff
@@ -4,4 +4,28 @@ data class OffenderDetail(
   val offenderId: Long,
   val otherIds: OtherIds,
   val offenderManagers: List<OffenderManager>,
+  val accessDenied: Boolean? = null,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**note:** ‏I've been reluctant to all the fields in this response in the past going with an approach of only surfacing the fields required for the work. The problem with this is we now have 20+ extra fields on the response, the vast majority will not be used. I suppose that now we know exactly what we get from the endpoint. Maybe need to discuss a common approach to this?

Reply:
removed all the unused fields.